mail_multicompany_force_company/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Email Multi Company Force Company Server",
+    "category": "Extra Tools",
+    "description": """
+       Email Multi Company Force Company Server:
+        This module forces to use the mail server that is configured for the company in outgoing mail server configuration.\n
+        If there is a mail server without a company assigned, it will be used by all companies.\n
+        In case there is a mail server without company and your company has assigned a mail server, you will use your company's mail server
```

Comment:
1. We don't use description in manifest, this should be in one of the `readme/*.rst` files. Check the OCA guidelines
2. The description on your PR was clearer, maybe you can use that one